test/test/console-log.tap

```diff
@@ -18,9 +18,10 @@ TAP version 13
         ok 4 - nested ok second
         1..4
     ok 2 - second ___/# time=[0-9.]+(ms)?/~~~
-ok 1 - nesting ___/# time=[0-9.]+(ms)?/~~~
 
 >>>> after second child
+ok 1 - nesting ___/# time=[0-9.]+(ms)?/~~~
+
```

Comment:
Oh, I see, so the issue here is that `nesting` was ending immediately upon the end of `nesting > second`, but now it's waiting until after the return of the `nesting` callback, because it returns a promise.

So, yeah, it's *relevant* (perhaps major version bump?) but yes, I see now, it is correct and necessarily implied by this feature.